src/Sylius/Behat/Context/Api/Shop/CheckoutContext.php

```diff
@@ -354,6 +355,27 @@ public function iProceededWithShippingMethod(ShippingMethodInterface $shippingMe
         $this->ordersClient->executeCustomRequest($request);
     }
 
+
+    /**
+     * @When /^I try to select inexistent "([^"]*)" payment method$/
+     */
+    public function iTryToSelectInexistentPaymentMethod(string $code): void
+    {
+        $paymentMethod = new PaymentMethod();
+
+        $request = Request::customItemAction(
+            'shop',
+            'orders',
+            $this->sharedStorage->get('cart_token'),
+            HTTPRequest::METHOD_PATCH,
+            sprintf('payments/%s', $this->getCart()['payments'][0]['id'])
+        );
+
+        $request->setContent(['paymentMethod' => $this->iriConverter->getItemIriFromResourceClass(\get_class($paymentMethod), ['code' => $code])]);
```

Comment:
`\get_class($paymentMethod)` -> `PaymentMethod::class`

Reply:
Looks same for me tbh. Convince me @Zales0123  why I should use it? 😄

Reply:
I am not @Zales0123 but I try to convince you :D because we should be consistent in Sylius, and even if both syntaxes do the same thing, second is used everywhere in Sylius :)

Reply:
RIP 7.x PHP
![Zrzut ekranu 2021-05-21 o 09 05 17](https://user-images.githubusercontent.com/35863747/119096276-ab899e80-ba13-11eb-8fb7-7b6636303309.png)